src/plugins/presentation_panel/public/panel_actions/customize_panel_action/custom_time_range_badge.tsx

```diff
@@ -16,12 +16,18 @@ import React from 'react';
 import { renderToString } from 'react-dom/server';
 
 import { UI_SETTINGS } from '@kbn/data-plugin/common';
-import { apiPublishesUnifiedSearch, EmbeddableApiContext } from '@kbn/presentation-publishing';
+import { PublishesUnifiedSearch, EmbeddableApiContext } from '@kbn/presentation-publishing';
 import { core } from '../../kibana_services';
 import { customizePanelAction } from '../panel_actions';
 
 export const CUSTOM_TIME_RANGE_BADGE = 'CUSTOM_TIME_RANGE_BADGE';
 
+const isApiCompatable = (
```

Comment:
nit: Maybe we should have a separate interface & type guard for `PublishesTimeRange` alongside `PublishesUnifiedSearch` - then we could use it instead of just checking for `api.timeRange$ === undefined`

Reply:
done in https://github.com/elastic/kibana/pull/178681/commits/f362c6413d3777844256611488195f6363fdd3e7